### ISSUE TITLE
Composite Noise and KSamplerAdvanced "no attribute" errors 

### ISSUE THREAD

**Issue by EgorkaSanyok** [ltdrdata/ComfyUI-Inspire-Pack#199]
Using the latest ComfyUI and Inspire pack, I am getting some "no attribute" errors in the following cases:

- Whenever connecting "Composite Noise (inspire)" to a node with noise input (such as "AddNoise", or even the "noise_opt" input of the "KsampleAdvanced (inspire)")

CompositeNoise //Inspire
'Noise_RandomNoise' object has no attribute 'clone'

![image1](https://github.com/user-attachments/assets/504b0ecf-9cd6-4eb7-88dd-0d4585d08f19)

- Whenever connecting any node that outputs noise (including the "RandomNoise (inspire)" node) to the "noise_opt" input of the "KsampleAdvanced (inspire)"

KSamplerAdvanced //Inspire
'Noise_RandomNoise' object has no attribute 'shape'

![image2](https://github.com/user-attachments/assets/c066bf2e-8e8b-4418-abff-b79cbffd9f3d)

Multiple other nodes could be affected, I've only tested the mentioned ones. Here are the error reports for both:
```
# ComfyUI Error Report
## Error Details
- **Node ID:** 15
- **Node Type:** CompositeNoise //Inspire
- **Exception Type:** AttributeError
- **Exception Message:** 'Noise_RandomNoise' object has no attribute 'clone'
## Stack Trace

  File "C:\ComfyUI\ComfyUI\execution.py", line 328, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\ComfyUI\ComfyUI\execution.py", line 203, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\ComfyUI\ComfyUI\execution.py", line 174, in _map_node_over_list
    process_inputs(input_dict, i)

  File "C:\ComfyUI\ComfyUI\execution.py", line 163, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\ComfyUI\ComfyUI\custom_nodes\ComfyUI-Inspire-Pack\inspire\prompt_support.py", line 779, in doit
    new_tensor = destination.clone()
                 ^^^^^^^^^^^^^^^^^
```
```
# ComfyUI Error Report
## Error Details
- **Node ID:** 20
- **Node Type:** KSamplerAdvanced //Inspire
- **Exception Type:** AttributeError
- **Exception Message:** 'Noise_RandomNoise' object has no attribute 'shape'
## Stack Trace

  File "C:\ComfyUI\ComfyUI\execution.py", line 328, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\ComfyUI\ComfyUI\execution.py", line 203, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\ComfyUI\ComfyUI\execution.py", line 174, in _map_node_over_list
    process_inputs(input_dict, i)

  File "C:\ComfyUI\ComfyUI\execution.py", line 163, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\ComfyUI\ComfyUI\custom_nodes\ComfyUI-Inspire-Pack\inspire\a1111_compat.py", line 195, in doit
    return (self.sample(*args, **kwargs)[0],)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\ComfyUI\ComfyUI\custom_nodes\ComfyUI-Inspire-Pack\inspire\a1111_compat.py", line 188, in sample
    return inspire_ksampler(model, noise_seed, steps, cfg, sampler_name, scheduler, positive, negative, latent_image,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\ComfyUI\ComfyUI\custom_nodes\ComfyUI-Inspire-Pack\inspire\a1111_compat.py", line 62, in inspire_ksampler
    if noise is not None and latent_image.shape[1] != noise.shape[1]:
                                                      ^^^^^^^^^^^
```

**Comment by ltdrdata** [comment]
The `NOISE` type used in `Inspire Pack` was created before the one in the ComfyUI core.

Because of compatibility with workflows that were already made with `Inspire Pack`, it cannot be changed recklessly.

Although the type name is the same `NOISE`, they are, in fact, different kinds of data. The core's `NOISE` is a noise generator, whereas the Inspire Pack's `NOISE` is a noisy latent.

This node should use the following:  


![image](https://github.com/user-attachments/assets/cb967dca-2eaa-4474-bfad-8d60e746ba2c)

**Comment by ltdrdata** [comment]
![composite-noise](https://github.com/user-attachments/assets/c528aeb3-98d9-4a07-af09-47a3aa06e23b)

It was my mistake. The `Random Noise (Inspire)` in the `Inspire Pack` is an extension of the core's `Random Noise` functionality.  

`Composite Noise` is related to the `Seed Explorer`. 
I got confused too, so I just renamed `NOISE` to `NOISE_IMAGE`.  
Fortunately, testing shows that there are no issues with the existing workflow.

https://github.com/ltdrdata/ComfyUI-extension-tutorials/blob/Main/ComfyUI-Inspire-Pack/workflow/seed-explorer.png
https://github.com/ltdrdata/ComfyUI-extension-tutorials/blob/Main/ComfyUI-Inspire-Pack/workflow/regional-seed.png
